docs/sp-add-ins/handle-events-in-sharepoint-add-ins.md

```diff
@@ -241,7 +241,7 @@ For a sample of an add-in event handler that does not use the handler delegation
 
 ## Remote event receivers in add-ins that support multiple security zones
 
-Some restrictions on how you design an add-in support multiple security zones and have a remote event receiver. For more information, see Knowledge Base article kb3135876 [You can't add a provider-hosted add-in to a SharePoint 2013 site in non-default zones](https://support.microsoft.com/help/3135876/you-can-t-add-a-provider-hosted-add-in-to-a-sharepoint-2013-site-in-no).
+Some restrictions on how you design an add-in support multiple security zones and have a remote event receiver. For more information, see Knowledge Base article kb3135876: You can't add a provider-hosted add-in to a SharePoint 2013 site in non-default zones.

```

Comment:
If the KB no longer exists, don't reference it in the docs. Right now it's telling the reader to go look at something with no link. Better to just remove it.